blackbox/docs/admin/ssl.txt

```diff
@@ -285,6 +285,15 @@ The common name (CN) should overlap with the CN of the server key
 generated in the first step. For example, ``ssl.crate.io`` overlaps
 with `*.crate.io`.
 
+.. NOTE::
+
+    In this step by step guide it is shown how to create a server certificate.
+    If you want to create a client certificate the steps are almost the same
+    with the exception of providing a common name that is equivalent to the
+    crate username as described in `client certificate authentication method`_.
+
+.. _client certificate authentication method: https://crate.io/docs/crate/reference/en/latest/admin/auth/methods.html#client-certificate-authentication-method
```

Comment:
There is already a link for that: https://github.com/crate/crate/blob/master/blackbox/docs/admin/auth/methods.txt#L100